src/app/list/list.component.scss

```diff
@@ -0,0 +1,36 @@
+* {
+    font-style: italic;
+}
+
+h3 {
+    margin-bottom: 60px;
+}
+
+button {
+    width: 200px;
+    padding: 10px;
+    margin-bottom: 15px;
+    margin-right: 10px;
+    font-size: large;
+    border-radius: 3px;
+}
+
+button:hover {
+    background-color: darkgreen;
```

Comment:
This adds a nice touch. If you want to add a layer of polish, try animating the color change so it fades in and out.